launcher/src/main/java/org/apache/spark/launcher/SparkLauncher.java

```diff
@@ -211,10 +211,10 @@ public SparkLauncher addSparkArg(String arg) {
    * Adds an argument with a value to the Spark invocation. If the argument name corresponds to
    * a known argument, the code validates that the argument actually expects a value, and throws
    * an exception otherwise.
-   * <p/>
+   * <p>
```

Comment:
So it turns out that javadoc in java 8 doesn't allow self-closing elements (```<br/>``` and ```<p/>```) any more:
http://stackoverflow.com/questions/26049329/javadoc-in-jdk-8-invalid-self-closing-element-not-allowed
http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#format

```<p>``` is the preferred seperator for a paragraph. So its is not HTML but Javadoc we are talking about. Sorry about the confusion.